templates/server/src/main/resources/archetype-resources/pom.xml

```diff
@@ -32,6 +32,14 @@
 
   <dependencyManagement>
     <dependencies>
+    <!-- overriding BOM of jackson -->
+    <dependency>
+        <groupId>com.fasterxml.jackson</groupId>
+        <artifactId>jackson-bom</artifactId>
+        <version>2.10.0.pr1</version>
```

Comment:
why are we using a different version here? Why are we not using a variable here? See also #118.